src/stories/components/MainMenu/styles.scss

```diff
@@ -3,52 +3,59 @@
 // @file
 // Main Menu
 
-// .main-menu {}
+.main-menu {
+  @include list-clean;
+}
+
+.main-menu__item {
+  margin: $spacing-2 0;
+}
 
 .menu-item__link {
-	display: flex;
-    cursor: pointer;
-    font-weight:normal; 
-    text-decoration: none;
-
-    &.active {
-        .menu-item__icon {
-            background: $teal;
-            fill: $white;
-            border-top-right-radius: 100px;
-            border-bottom-right-radius: 100px;
-        }
-
-        .menu-item__text {
-            color: $teal-dark;
-        }
+  align-items: center;
```

Comment:
You can center icon and text by adding `align-items: center` to the wrapper link.